services/src/unit-test/java/uk/gov/moj/sdt/services/messaging/MessageWriterTest.java

```diff
@@ -110,87 +111,73 @@ public void testQueueMessageWithEmptyTargetApp() {
      * to queue name map is not valid
      */
     @Test
-    public void testQueueMessageWithInvalidTarget() {
+    void testQueueMessageWithInvalidTarget() {
         // Setup finished, now tell the mock what to expect.
         final ISdtMessage sdtMessage = new SdtMessage();
         sdtMessage.setSdtRequestReference("Test");
 
-        messageSender.sendMessage("UnitTestQueue", sdtMessage);
-        EasyMock.expectLastCall();
-
-        // Get ready to call the mock.
-        replay(messageSender);
+        messageSender.sendMessage(UNIT_TEST_QUEUE, sdtMessage);
```

Comment:
This test will cause a failure before messageSender.sendMessage() is invoked, so this interaction with the mocked object will not occur. Line should be removed and messageSender verified for no invocation of sendMessage() either in the catch block or after it.

Reply:
we don't need to call messageSender.sendMessage("UnitTestQueue.DLQ", sdtMessage);

**removed this one now**

**and verifying like below:**

        verify(messageSender, times(0)).sendMessage(any(), any());

Reply:
Confirmed sendMessage() line removed and check made to verify that it hasn't been invoked.